bigchaindb/backend/query.py

```diff
@@ -259,15 +286,15 @@ def get_genesis_block(connection):
 
 
 @singledispatch
-def get_last_voted_block(connection, node_pubkey):
+def get_last_voted_block_id(connection, node_pubkey):
     """Get the last voted block for a specific node.
 
     Args:
         node_pubkey (str): base58 encoded public key.
 
     Returns:
-        The last block the node has voted on. If the node didn't cast
-        any vote then the genesis block is returned.
+        The last block id the node has voted on. If the node didn't cast
+        any vote then the genesis block id is returned.
```

Comment:
Really small detail, but it seems to me that it would be more accurate to write something like:

> The id of the last block the node has voted on.

Otherwise, "the last block id the node has voted on" sounds like the vote is being done on the id rather than on the block itself.

@ttmc thoughts?